ui/src/app/features/collection-manager-subject-left/collection-manager-subject-left.component.scss

```diff
@@ -0,0 +1,88 @@
+.main {
+  padding: 24px 8px 0px 8px;
+  //min-height: 425px;
+  //height: 425px;
+  //overflow-y: scroll;
+  .button {

```

Comment:
as mentioned above most all button styles should come from our custom angular material theme, you should not define it like this.